```

'WorkspaceTests.testArtifactDownloadHappyPath' started at 2021-02-24 06:35:06.861
/home/buildnode/jenkins/workspace/swift-PR-Linux/branch-main/swiftpm/Tests/WorkspaceTests/WorkspaceTests.swift:4402: error: WorkspaceTests.testArtifactDownloadHappyPath : XCTAssertEqual failed: ("[<AbsolutePath:"/tmp/ws/.build/artifacts/A">, <AbsolutePath:"/tmp/ws/.build/artifacts/B">]") is not equal to ("[<AbsolutePath:"/tmp/ws/.build/artifacts/A">, <AbsolutePath:"/tmp/ws/.build/artifacts/A">, <AbsolutePath:"/tmp/ws/.build/artifacts/B">]") 
```

Hm.

--- a/stdlib/public/core/Sort.swift
+++ b/stdlib/public/core/Sort.swift
@@ -246,7 +246,7 @@ extension MutableCollection where Self: RandomAccessCollection {
   public mutating func sort(
     by areInIncreasingOrder: (Element, Element) throws -> Bool
   ) rethrows {
-    let didSortUnsafeBuffer =
+    let didSortUnsafeBuffer: Void? =
       try withContiguousMutableStorageIfAvailable { buffer in
         try buffer._stableSortImpl(by: areInIncreasingOrder)
       }

